Those_are_the_names.py

```diff
@@ -0,0 +1,13 @@
+def full_names(first_name_list, last_name_list, min_len=0):
```

Comment:
1. Documentation.
2. We encourage you to write oneliners, but sometimes it makes it a bit harder for reading.
Try to simplify that.